sdk/identity/azure-identity-v2/src/main/java/com/azure/identity/v2/ManagedIdentityCredential.java

```diff
@@ -84,8 +83,6 @@ public final class ManagedIdentityCredential implements TokenCredential {
      * @param miClientOptions the options for configuring the identity client.
```

Comment:
also cleanup the javadoc here since this is an options bag now (ie, you don't pass clientID here.)